gpytorch/kernels/distributional_input_kernel.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+from torch.distributions import Normal, kl_divergence
+
+from .kernel import Kernel
+
+
+def _symmetrized_kl(dist1, dist2, num_dims=2, eps=1e-8):
```

Comment:
maybe add some docstring here to make clear that this assumes that the inputs are assumed MVNs with diagonal covars